src/rng.rs

```diff
@@ -409,27 +415,31 @@ macro_rules! impl_fill {
         }
 
         impl Fill for [Wrapping<$t>] {
-            #[inline(never)]
             fn fill<R: Rng + ?Sized>(&mut self, rng: &mut R) {
                 if self.len() > 0 {
-                    rng.fill_bytes(self.as_mut_bytes());
+                    rng.fill_bytes(unsafe {
+                        slice::from_raw_parts_mut(self.as_mut_ptr()
+                            as *mut u8,
+                            self.len() * mem::size_of::<$t>()
+                        )
+                    });
                     for x in self {
-                    *x = Wrapping(x.0.to_le());
+                        *x = Wrapping(x.0.to_le());
                     }
                 }
             }
         }
     };
     ($t:ty, $($tt:ty,)*) => {
-        impl_fill!($t);
+        unsafe_impl_fill!($t);
         // TODO: this could replace above impl once Rust #32463 is fixed
-        // impl_fill!(Wrapping<$t>);
-        impl_fill!($($tt,)*);
+        // unsafe_impl_fill!(Wrapping<$t>);
+        unsafe_impl_fill!($($tt,)*);
     }
 }
 
-impl_fill!(u16, u32, u64, u128,);
-impl_fill!(i8, i16, i32, i64, i128,);
+unsafe_impl_fill!(u16, u32, u64, u128,);
+unsafe_impl_fill!(i8, i16, i32, i64, i128,);
```

Comment:
Safety comments?

You may want to copy from [what we've done in zerocopy](https://github.com/google/zerocopy/blob/dccbbcd8714eebeeac794ca5dd1ba1cbecc09ea2/src/impls.rs#L33-L47) (specifically for `FromBytes`).

Reply:
I added a very minimal comment.